google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -798,10 +798,16 @@ class SpannerDialect(DefaultDialect):
     supports_sequences = True
     sequences_optional = False
     supports_identity_columns = True
-    supports_native_enum = True
     supports_native_boolean = True
     supports_native_decimal = True
     supports_statement_cache = True
+    # Spanner uses protos for enums. Creating a column like
+    # Column("an_enum", Enum("A", "B", "C")) will result in a String
+    # column. Setting supports_native_enum to False means SQLAlchemy
+    # will generate check constraints to enforce the enum values
```

Comment:
I'm not sure this change is enough to actually achieve this. Would you otherwise mind adding a test that shows that it does?

I tried the same here, but it does not seem to make any difference: https://github.com/googleapis/python-spanner-sqlalchemy/pull/698/files